app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,74 @@
+class MoviesController < ApplicationController
+  before_action :set_movie, only: [:show, :edit, :update, :destroy]
+
+  # GET /movies
+  # GET /movies.json
+  def index
+    @movies = Movie.all
+  end
+
+  # GET /movies/1
+  # GET /movies/1.json
+  def show
```

Comment:
Same here, the `#show` method is essential to every app.
Even if the lesson/instruction did not mention/ask u to add it, it's important to always have it.
The variable `@movie` is defined in the `private method block`, so all u need to do is pass the param.